--- php-7.3.5/Zend/zend_alloc.c~	2019-04-30 08:37:26.000000000 +0000
+++ php-7.3.5/Zend/zend_alloc.c	2019-05-08 02:35:38.935917600 +0000
@@ -477,6 +477,9 @@
 #endif
 		return NULL;
 	}
+	if (zend_mm_use_huge_pages && size == ZEND_MM_CHUNK_SIZE) {
+		madvise (ptr, size , MADV_HUGEPAGE);
+	}
 	return ptr;
 #endif
 }
--- php-7.3.5/ext/opcache/shared_alloc_mmap.c~	2019-04-30 08:37:18.000000000 +0000
+++ php-7.3.5/ext/opcache/shared_alloc_mmap.c	2019-05-08 02:37:04.365916410 +0000
@@ -67,6 +67,7 @@
 		*error_in = "mmap";
 		return ALLOC_FAILURE;
 	}
+	madvise (shared_segment->p, requested_size , MADV_HUGEPAGE);
 
 	shared_segment->pos = 0;
 	shared_segment->size = requested_size;
--- php-7.3.5/Zend/zend_alloc.c~	2019-05-08 02:35:38.000000000 +0000
+++ php-7.3.5/Zend/zend_alloc.c	2019-05-08 14:57:41.625297719 +0000
@@ -191,7 +191,7 @@
 typedef struct  _zend_mm_chunk     zend_mm_chunk;
 typedef struct  _zend_mm_huge_list zend_mm_huge_list;
 
-int zend_mm_use_huge_pages = 0;
+int zend_mm_use_huge_pages = 1;
 
 /*
  * Memory is retrived from OS by chunks of fixed size 2MB.
